### DISCUSSION TITLE
Dynamic imagepolicy comment

### DISCUSSION THREAD

**Discussion by lapwat** [fluxcd/flux2#1759] [Q&A] [answered]
I would like to have a dynamic ImagePolicy comment, something like:

```yaml
image: "myimage:{{ .Release.Name }" # {"$imagepolicy": "mynamespace:{{ .Release.Name }}"}
```

Is it possible to achieve ?

The problem here is that helm omits the comments when generating yaml files.

**Comment by hiddeco** [comment, marked as answer]
Even if Helm would have been able to add the comments, this would not work because the image automation object has no way to update the value in the YAML templates of the chart.

Instead, you should add the comment to the `HelmRelease` which is sourced and applied from a `GitRepository`, so that the image automation logic can update the manifest in Git.

You can find more information about this here: https://fluxcd.io/docs/guides/image-update/#configure-image-update-for-custom-resources

**Reply by lapwat** [reply]
Thank you I will try that way
